cdap-data-fabric/src/main/java/co/cask/cdap/data/runtime/DataFabricDistributedModule.java

```diff
@@ -63,44 +50,4 @@ public void configure() {
     install(new TransactionModules().getDistributedModules());
 
   }
-
-  /**
-   * Provides implementation of {@link ThriftClientProvider} based on configuration.
-   */
-  @Singleton
-  private static final class ThriftClientProviderSupplier implements Provider<ThriftClientProvider> {

```

Comment:
Can you use `TransactionClientModule` from Tephra instead? We don't need to change this module now. For the tool purpose, doesn't need to choose client based on cConf.